DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Configuration/RegistrationPromptsController.cs

```diff
@@ -444,10 +444,11 @@ private void SetTotalAnswersLengthTooLongError(RegistrationPromptAnswersViewMode
                 return;
             }
 
-            var remainingLength = 4000 - (model.OptionsString?.Length - 2 ?? 0);
+            var remainingLength = 4000 - model.OptionsString.Length;

```

Comment:
I can't comment it against the actual line of the code, but we can probably get rid of the `|| model.OptionsString.Length < 2` on both places with this logic